addons/web/controllers/dataset.py

```diff
@@ -13,17 +13,27 @@
 _logger = logging.getLogger(__name__)
 
 
+def _call_kw_readonly(registry, request):
+    params = request.get_json_data()['params']
+    model_class = registry[params['model']]
+    method_name = params['method']
+    for cls in model_class.mro():  # TODO this is maybe not the most efficient way to do that, we could maybe set it on the method during setup
```

Comment:
Remove the comment, it only adds noise IMHO.
If this causes performance issues, we will find it and optimize it.
Our preliminary tests showed that this does not cause performance issues.